### ISSUE TITLE
Improving login form

### ISSUE THREAD

**Issue by Manishar55** [harshalhonde21/EcommerceSpectastyle#142]
### Describe the feature

Hii @harshalhonde21 
The login form not looking good, it is very small I want to make it little bigger and more attractive
Can you please assign this task to me under gssoc24

### Add ScreenShots

![Screenshot 2024-05-12 232126](https://github.com/harshalhonde21/EcommerceSpectastyle/assets/154615478/2ede4ab6-13d6-4c24-94ac-3b745cad49d3)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by mihirpatel07a** [comment]
yes i have very impressive design of login form please give assign me this issue @harshalhonde21  @SaranshBangar

**Comment by SaranshBangar** [comment]
@mihirpatel07a why am I tagged?

**Comment by mihirpatel07a** [comment]
Sorry I felt that you are project admin

**Comment by harshalhonde21** [comment]
Login form is great as theme of website no need of this but you can create and give me sample and raise the PR

**Comment by mihirpatel07a** [comment]
Ok, will do that.
